src/config.js

```diff
@@ -1,17 +1,18 @@
 import { merge } from 'lodash'
 
 // https://personal.sron.nl/~pault/
-const tolMutedQualitative = [
-  '#332288', // indigo
-  '#88CCEE', // cyan
-  '#44AA99', // teal
-  '#117733', // green
-  '#999933', // olive
-  '#DDCC77', // sand
-  '#CC6677', // rose
-  '#882255', // wine
-  '#AA4499', // purple
-  '#DDDDDD' // grey
+// light: muted qualitative, dark: light qualitative (w/ muted purple)
+const defaultColours = [
+  { light: '#332288', /* indigo */ dark: '#77AADD' /* light blue */ },
```

Comment:
These first three dark colours, as shown here, don't have much contrast. And having only, say, the first 3 in use might not be uncommon. Can we shuffle the full set up as far as possible so that the first N are fairly far apart?